pkgs/by-name/gr/grooveauthor-bin/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  autoPatchelfHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  wrapGAppsHook3,
+  alsa-lib,
+  dotnetCorePackages,
+  gtk3,
+  libGL,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "grooveauthor-bin";
```

Comment:
We do not name binaries with bin at the end like Arch Linux. Pleas set meta.sourceProvenance accordingly instead.